src/bin/gtfs2ntfs.rs

```diff
@@ -39,7 +39,7 @@ struct Opt {
 
     /// config file
     #[structopt(short = "c", long = "config", parse(from_os_str))]
-    config_path: Option<PathBuf>,
+    config_path: PathBuf,
```

Comment:
We need to be able to use navitia_model without prefix and without doing that, creating some default objects.

Reply:
For the GTFS->NTFS conversion, I asked for a systematic prefix with a pattern <prefix>:<object_id>. If I understand, you ask for an optional config file that would create default contributor and dataset. How about a predefined prefix like "default_prefix" ?
An empty <prefix> would imply also removing the ":" separator, and would add quite complexity to the code. 
If you're all ok with that, I'll specify the default values to avoid refacto :D